atlp-rwanda/stackup2-barefoot-frontend
# #171380695 user login

#### What does this PR do?

- user should be able to login

#### Description of Task to be completed?
- Install required libraries
- Create UserLogin component

#### How should this be manually tested?
- In your terminal, run `git clone https://github.com/Stackup-Rwanda/stackup2-barefoot-frontend.git` to clone this repo

- Run `git fetch origin ft-user-login-171380695

- Run `git checkout ft-user-login-171380695` to switch to the fetched branch above

- Run `yarn install` to install all dependencies

- Start the development server by running `yarn dev`.

- The server should launch the app in your default browser at `localhost:{{PORT}}`
- Click the login button, you will see the login form
- try to login with the user in database 

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
  [#171380695](https://www.pivotaltracker.com/story/show/171380695)
